contracts/optimistic-ethereum/OVM/precompiles/OVM_ETH.sol

```diff
@@ -11,7 +11,12 @@ import { iOVM_BaseCrossDomainMessenger } from "../../iOVM/bridge/iOVM_BaseCrossD
 
 /**
  * @title OVM_ETH
- * @dev L2 CONTRACT (COMPILED)
+ * @dev The ETH predeploy provides an ERC20 interface for ETH deposited to Layer 2. Note that 
+ * unlike on Layer 1, Layer 2 accounts do not have a balance field.
```

Comment:
nit: we do technically have a balance field, just left it untouched in geth.  However, there is no way for it to have a nonzero value.